.github/workflows/release.yaml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: taiki-e/create-gh-release-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  upload-assets:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: taiki-e/upload-rust-binary-action@v1
+        with:
+          bin: cksfv
+          token: ${{ secrets.GITHUB_TOKEN }}
+          checksum: sha256
```

Comment:
Please also include the `README.md`, `COPYING`, and `CHANGELOG.md` files to the binary releases, this can be done in `taiki-e/upload-rust-binary-action@v1` with:
```yaml
with:
   # ... #
   include: CHANGELOG.md,COPYING,README.md
```

Reply:
Updated